From 2482cb035ec2a894a5910d4a47ad3ca19fff511e Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@altera.com>
Date: Tue, 28 May 2013 21:46:48 -0500
Subject: [PATCH 247/254] arm: socfpga: Remove FPGA entries from socfpga.dtsi

Upstream: git://git.rocketboards.org/linux-socfpga.git

Remove FPGA IPs from device tree base and board files. The FPGA IPs
entries should be placed in a separate dts file.

Signed-off-by: Dinh Nguyen <dinguyen@altera.com>

Conflicts:

	arch/arm/boot/dts/socfpga.dtsi
(cherry picked from commit d1592c1487a7c0e3ebe13b839080d32f6d2229ea)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/boot/dts/socfpga.dtsi         |   14 --------------
 arch/arm/boot/dts/socfpga_cyclone5.dts |   20 --------------------
 2 files changed, 0 insertions(+), 34 deletions(-)

diff --git a/arch/arm/boot/dts/socfpga.dtsi b/arch/arm/boot/dts/socfpga.dtsi
index 6c6429c..f3563e6 100644
--- a/arch/arm/boot/dts/socfpga.dtsi
+++ b/arch/arm/boot/dts/socfpga.dtsi
@@ -66,20 +66,6 @@
 		interrupt-parent = <&intc>;
 		ranges;
 
-		agpio0: gpio@0xc0000000 {
-			compatible = "altr,pio-1.0";
-			/* Register base 0xff200000 is for a light-weight bridge */
-			reg = <0xff210040 0x10>;
-			width = <32>;
-			/* There are 64 interrupts from the FPGA start at 72, so 45 has to be wrong */
-			interrupts = <0 45 4>;
-			interrupt-controller;
-			#interrupt-cells = <1>;
-			gpio-controller;
-			#gpio-cells = <2>;
-			level_trigger = <0>;
-		};
-
 		amba {
 			compatible = "arm,amba-bus";
 			#address-cells = <1>;
diff --git a/arch/arm/boot/dts/socfpga_cyclone5.dts b/arch/arm/boot/dts/socfpga_cyclone5.dts
index 73cf8d4..d640b80 100644
--- a/arch/arm/boot/dts/socfpga_cyclone5.dts
+++ b/arch/arm/boot/dts/socfpga_cyclone5.dts
@@ -173,26 +173,6 @@
 
 		leds {
 			compatible = "gpio-leds";
-			fpga0 {
-				label = "fpga_led0";
-				gpios = <&agpio0 0 1>;
-			};
-
-			fpga1 {
-				label = "fpga_led1";
-				gpios = <&agpio0 1 1>;
-			};
-
-			fpga2 {
-				label = "fpga_led2";
-				gpios = <&agpio0 2 1>;
-			};
-
-			fpga3 {
-				label = "fpga_led3";
-				gpios = <&agpio0 3 1>;
-			};
-
 			hps0 {
 				label = "hps_led0";
 				gpios = <&gpio1 15 1>;
-- 
1.7.5.4

